azuredeploy.json

```diff
@@ -567,6 +567,13 @@
                 "description": "Virtual network gateway vpn type"
             },
             "type": "string"
+        },
+        "loadBalancerSku": {
+            "defaultValue": "Basic",
+            "metadata": {
+                "description": "Loadbalancer SKU"
```

Comment:
Seeing that the LB SKU is used as the public IP SKU below (which definitely appears to be an Azure requirement), it'd be great if you add that aspect as part of this description.